tests/debugger/do_display.cmd

```
@@ -0,0 +1,13 @@
+b a
+b b
+b c
+b d
+display
```

I think display without argument is pointless, I would throw an error.